enforcer-rules/src/main/java/org/apache/maven/enforcer/rules/BannedRepositories.java

```diff
@@ -96,18 +96,18 @@ public void execute() throws EnforcerRuleException {
     }
 
     // ----------------------------------------------------------------------
-    // Protected methods
+    // Package methods
```

Comment:
Changing visibility is an incompatibility.  Are we sure there are no custom rules overriding those methods ?

Reply:
The class is final. That's why IntelliJ flagged these.

Reply:
yes for final class protected has no sense
maybe - private

Reply:
I tried private but these methods are used in the test classes.